### DISCUSSION TITLE
Code Not Conforming To PEP8 Standards

### DISCUSSION THREAD

**Discussion by Xceptions** [encode/django-rest-framework#7786] [Question & Answer] [answered]
Hi all,

I have been an avid user of DRF in my projects but only decided to start contributing to it just recently. And the first place I decided to check, of course, was for typos and non-conforming codes

The contributing page says

**Changes should broadly follow the PEP8 style conventions, and we recommend you set up your editor to automatically indicate non-conforming styles.**

However, when I pasted the code from the serializers.py page into the [PEP8](http://pep8online.com/) online checker, it brought back a lot of *line too long* errors. E.g.

    line too long (88 > 79 characters)

Is this to be ignored or do I go-ahead to work on shortening the codes on each line?
Thanks in advance.

**Comment by carltongibson** [comment, marked as answer]
Hi @Xceptions — yes, that's ignored. See the `flake8` config here: 

https://github.com/encode/django-rest-framework/blob/ec29ff8a8013dd3383344bb78eea479d025e2a87/setup.cfg#L8-L10

**Reply by Xceptions** [reply]
Okay. Thanks for the reply
